data/mllib/sample_libsvm_data_with_unique_column.txt

```diff
@@ -0,0 +1,270 @@
+1 1:0.708333 2:1 3:1 4:-0.320755 5:-0.105023 6:-1 7:1 8:-0.419847 9:-1 10:-0.225806 12:1 13:-1 14:1
```

Comment:
This file is for review only, to show an example on a larger data set. Will remove if merged.